pkg/controller/flow/reconcile.go

```diff
@@ -350,3 +357,41 @@ func (r *reconciler) NewControllerRef(flow *v1alpha1.Flow) *metav1.OwnerReferenc
 	revRef.Controller = &isController
 	return revRef
 }
+
+const (
+	// controllerConfigMapName is the name of the configmap in the eventing
+	// namespace that holds the configuration for this controller.
+	controllerConfigMapName = "flow-controller-config"
+
+	// defaultClusterBusConfigMapKey is the name of the key in this controller's
+	// ConfigMap that contains the name of the default cluster bus for the flow
+	// controller to use.
+	defaultClusterBusConfigMapKey = "default-cluster-bus"
+
+	// fallbackClusterBusName is the name of the cluster bus that will be used
+	// for flows if the controller's configmap does not exist or does not
+	// contain the 'default-cluster-bus' key.
+	fallbackClusterBusName = "stub"
+)
+
+// getDefaultClusterBusName returns the value of the 'default-cluster-bus' key in
+// the knative-system/flow-controller-config configmap or an error. If the
+// 'default-cluster-bus' key is not set, it returns the default value "stub".
```

Comment:
Is this something which should be handled through an Informer in the future (it's fine to leave this as a TODO)?

Reply:
Do you need to update https://github.com/knative/eventing/blob/master/config/200-clusterrole.yaml ?

Reply:
No, it turns out that we don't care about roles: https://github.com/knative/eventing/blob/master/config/201-clusterrolebinding.yaml (`cluster-admin`)

Reply:
The controller-runtime client handles maintaining informers automatically - pretty neat huh?